build-tools/packages/build-tools/packlist.txt

```diff
@@ -0,0 +1,266 @@
+dist/fluidBuild/tasks/leaf/ts2EsmTask.d.ts
+dist/fluidBuild/tasks/leaf/ts2EsmTask.d.ts.map
+dist/fluidBuild/tasks/leaf/ts2EsmTask.js
+dist/fluidBuild/tasks/leaf/ts2EsmTask.js.map
+dist/fluidBuild/tasks/leaf/tscTask.d.ts
+dist/fluidBuild/tasks/leaf/tscTask.d.ts.map
+dist/fluidBuild/tasks/leaf/tscTask.js
+dist/fluidBuild/tasks/leaf/tscTask.js.map
+dist/fluidBuild/tasks/leaf/webpackTask.d.ts
+dist/fluidBuild/tasks/leaf/webpackTask.d.ts.map
+dist/fluidBuild/tasks/leaf/webpackTask.js
+dist/fluidBuild/tasks/leaf/webpackTask.js.map
+dist/fluidBuild/tasks/task.d.ts
+dist/fluidBuild/tasks/task.d.ts.map
+dist/fluidBuild/tasks/task.js
+dist/fluidBuild/tasks/task.js.map
+dist/fluidBuild/tasks/taskFactory.d.ts
+dist/fluidBuild/tasks/taskFactory.d.ts.map
+dist/fluidBuild/tasks/taskFactory.js
+dist/fluidBuild/tasks/taskFactory.js.map
+dist/fluidBuild/tasks/workers/eslintWorker.d.ts
+dist/fluidBuild/tasks/workers/eslintWorker.d.ts.map
+dist/fluidBuild/tasks/workers/eslintWorker.js
+dist/fluidBuild/tasks/workers/eslintWorker.js.map
+dist/fluidBuild/tasks/workers/tscWorker.d.ts
+dist/fluidBuild/tasks/workers/tscWorker.d.ts.map
+dist/fluidBuild/tasks/workers/tscWorker.js
+dist/fluidBuild/tasks/workers/tscWorker.js.map
+dist/fluidBuild/tasks/workers/worker.d.ts
+dist/fluidBuild/tasks/workers/worker.d.ts.map
+dist/fluidBuild/tasks/workers/worker.js
+dist/fluidBuild/tasks/workers/worker.js.map
+dist/fluidBuild/tasks/workers/workerPool.d.ts
+dist/fluidBuild/tasks/workers/workerPool.d.ts.map
+dist/fluidBuild/tasks/workers/workerPool.js
+dist/fluidBuild/tasks/workers/workerPool.js.map
+dist/fluidTsc/fluidTsc.d.ts
+dist/fluidTsc/fluidTsc.d.ts.map
+dist/fluidTsc/fluidTsc.js
+dist/fluidTsc/fluidTsc.js.map
+dist/index.d.ts
+dist/index.d.ts.map
+dist/index.js
+dist/index.js.map
+dist/type-test-generator/compatibility.d.ts
+dist/type-test-generator/compatibility.d.ts.map
+dist/type-test-generator/compatibility.js
+dist/type-test-generator/compatibility.js.map
+dist/type-test-generator/main.d.ts
+dist/type-test-generator/main.d.ts.map
+dist/type-test-generator/main.js
+dist/type-test-generator/main.js.map
+dist/typeValidator/changeToNamedExports.d.ts
+dist/typeValidator/changeToNamedExports.d.ts.map
+dist/typeValidator/changeToNamedExports.js
+dist/typeValidator/changeToNamedExports.js.map
+dist/typeValidator/testGeneration.d.ts
+dist/typeValidator/testGeneration.d.ts.map
+dist/typeValidator/testGeneration.js
+dist/typeValidator/testGeneration.js.map
+dist/typeValidator/typeData.d.ts
+dist/typeValidator/typeData.d.ts.map
+dist/typeValidator/typeData.js
+dist/typeValidator/typeData.js.map
+dist/typeValidator/typeDecomposition.d.ts
+dist/typeValidator/typeDecomposition.d.ts.map
+dist/typeValidator/typeDecomposition.js
+dist/typeValidator/typeDecomposition.js.map
+dist/typeValidator/validatorUtils.d.ts
+dist/typeValidator/validatorUtils.d.ts.map
+dist/typeValidator/validatorUtils.js
+dist/typeValidator/validatorUtils.js.map
+src/common/commonOptions.ts
```

Comment:
Naughty build-tools, src files shouldn't be here 😝 .

Reply:
We include source in all our packages. :)

Reply:
Oh right it's src/test/ we exclude in most of them... got confused because I saw build-cli doesn't (and it's one of the few that uses package.json's  `files`, I think?) but I guess that one doesn't need it as much. src/ being there is part of what makes source mapping work while debugging, right?

Reply:
> src/ being there is part of what makes source mapping work while debugging, right?

Unclear. :) @dzearing opened a bug at one point suggesting we remove the src, but @christiango was concerned that might affect debugging. The source maps _should_ be enough, though. I've never had time to test it.

Reply:
I'm not 100% sure of the context here, but I think the concern we had in the past with not having src in npm packages is that it would break "go to definition" / "go to source" in vscode for repos that consume FF